src/spec.md

```diff
@@ -28,7 +28,8 @@ Send a status update to all peers.
 - `update`: an object with the following properties:  
     - `update.payload`: string, number, boolean, array, object or `null`.
        MUST NOT be `undefined`.
-       Binary data buffers as used in `File`, `Blob`, `Int*Array` etc. are skipped;
+       Everything that is not JSON serialisable will be skipped,
```

Comment:
we (and most other tech docs) usually prefer AE over BE - therefore the `z`:

```suggestion
       Everything that is not JSON serializable will be skipped,
```

(but even in BE, the `z` variation seems to be dominant, eg. https://sapling.ai/usage/serialization-vs-serialisation)